### PR TITLE
Web3: Show recipient and value on Transaction confirmation

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-web3/src/ui/react/Confirmation/TransactionSpeed.tsx [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
+export const TransactionSpeed = ({gasModes, selectedValue, onChange}: TransactionSpeedProps) => {
+  return (
+    <Row>
+      {gasModes.map(({name, usdAmount}) => (
+        <RadioButton
+          key={name}
+          value={name}
+          selectedValue={selectedValue}
+          onChange={onChange}
+          time="1 day"
```
Comment (JustynaBroniszewska) [inline universal-login-web3/src/ui/react/Confirmation/TransactionSpeed.tsx]: Is 1 day correct for each gasMode?

File: universal-login-web3/src/ULWeb3Provider.ts [hunk 2]
```diff
@@ -165,7 +165,7 @@ export class ULWeb3Provider implements Provider {
     this.uiController.finishOnboarding();
   }
 
-  private async ensureWalletIsDeployed() {
+  private async deployWalletIfNotExists() {
```
Comment (JustynaBroniszewska) [inline universal-login-web3/src/ULWeb3Provider.ts]: Rename proposals:
- `deployIfNoWallet`
- `deployIfNoWalletDeployed`
- `deployIfNoWalletExist`

Comment (xitronix) [reply universal-login-web3/src/ULWeb3Provider.ts]: Done

File: universal-login-web3/src/ui/react/Confirmation/TransactionSpeed.tsx (hunk 1 above)
Comment (xitronix) [reply universal-login-web3/src/ui/react/Confirmation/TransactionSpeed.tsx]: It is mock up, so i just write value. We don't have any service which calculate it. Which value do you suggest write here?

File: universal-login-web3/src/ui/react/Confirmation/TransactionConfirmation.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+                <DataLabel>Send to:</DataLabel>
+                <Address>{transaction.to}</Address>
+              </Row>
+              <Row>
+                <DataLabel>Value:</DataLabel>
+                <ValueRow>
+                  <Highlighted>
+                    <Value>{utils.formatEther(transaction.value)} ETH</Value>
+                  </Highlighted>
+                  <Value>{mode.usdAmount}</Value>
```
Comment (marekkirejczyk) [inline universal-login-web3/src/ui/react/Confirmation/TransactionConfirmation.tsx]: If not working better hide

**Pull request merged**